lib/hydra/derivatives/processors/full_text_extraction.rb

```diff
@@ -0,0 +1,59 @@
+module Hydra::Derivatives::Processors
+  # Extract the full text from the content using Solr's extract handler
+  class FullTextExtraction < Processor
+
+    def process
```

Comment:
This method seems a bit mysterious to me, since it is the only public method with no docs.